R/derive_seq.R

```diff
@@ -0,0 +1,68 @@
+#' Derive the sequence number (`--SEQ`) variable
+#'
+#' @description
+#' [derive_seq()] creates a new identifier variable: the sequence number
+#' (`--SEQ`).
+#'
+#' This function adds a newly derived variable to `tgt_dat`, namely the sequence
+#' number (`--SEQ`) whose name is the one provided in `tgt_var`. An integer
+#' sequence is generated that uniquely identifies each record within the domain.
+#'
+#' Prior to the derivation of `tgt_var`, the data frame `tgt_dat` is sorted
+#' according to grouping variables indicated in `rec_vars`.
+#'
+#' @param tgt_dat The target dataset, a data frame.
+#' @param tgt_var The target SDTM variable: a single string indicating the name
+#'   of the sequence number (`--SEQ`) variable, e.g. `"DSSEQ"`. Note that
```

Comment:
@edgar-manukyan: Was this comment for me, or perhaps for @rammprasad ?

Reply:
I was just reading the function header and it says:
>  Note that  supplying a name not ending in `"SEQ"` will raise a warning.

The I supplied a name that did not end with SEQ expecting a warning, but there was no warning.

Reply:
Okay, so you agree that a warning is appropriate, correct?

Reply:
I think that would make sense and if we are documenting it then we better do it 👍

Reply:
Yeah, I had initially planned to do it, so I documented it but then I did not implement it because I felt it would be better to ask you guys... And then I forgot and the docs remained...